crates/autopilot/src/database/competition.rs

```diff
@@ -16,8 +16,7 @@ use {
 #[derive(Clone, Debug)]
 pub enum ExecutedFee {
     Solver(U256),
-    /// Optional because, for partially fillable limit orders, surplus fee is
-    /// unknown until the transaction is mined.
+    /// Fee for limit orders.
     Surplus(Option<U256>),
```

Comment:
The old comment says this is optional because of *reason*. Is that reason no longer valid? Then why is it still optional?

Reply:
Agree, that the inner type can probably be dropped altogether since it should now always be `None`.

Reply:
Indeed, dropped it. 
The variant itself should be kept to indicate that the order has that type of fee when saving order executions to db.